### ISSUE TITLE
12cdetect not showing connection

### ISSUE THREAD

**Issue by TheClergy** [Freenove/Freenove_4WD_Smart_Car_Kit_for_Raspberry_Pi#33]
When I run the the i2cdetect command it is not showing the board connection
![i2c detect issue](https://user-images.githubusercontent.com/102489746/163829835-a5cc0333-1c32-41fe-a9ff-90a9ea609392.png)

**Comment by MallocArray** [comment]
Did you turn on both power switches?  One controls the Raspberry Pi power and one to the rest of the board such as motors and servos.

**Comment by TheClergy** [comment]
Both buttons were on but if was also plugged in to power would that change anything

**Comment by janrutger** [comment]
No, you do not damge the board, to run the test, you need also the battery charged and connected.
